learn/configuration/configuring_index_settings.mdx

```diff
@@ -28,37 +28,39 @@ Using the menu on the left-hand side, click on "Attributes":
+Meilisearch will display a pop-up window asking you to confirm you want to remove the attribute from the searchable attributes list. Click on "Yes, remove attribute":
 
-![The "Searchable attributes" configuration section showing two buttons at the bottom of the screen: "Cancel changes" and "Save".](/assets/images/cloud-index-settings/06-save.png)
+![A pop-up window over the index settings interface. It reads: "Are you sure you want to remove the attribute id?" Below it are two buttons: "Cancel" and "Yes, remove attribute".](/assets/images/cloud-index-settings/06-searchable-attributes-confirm-deletion.png)
 
-The button icon will change to a spinning wheel to indicate Meilisearch is processing the changes. Depending on the size of your dataset and the setting you have edited, this might take some time:
+Most updates to an index's settings will cause Meilisearch to re-index all its data. Wait a few moments until this operation is complete. You are not allowed to update any index settings during this time.
 
-![The "Searchable attributes" configuration section showing two buttons at the bottom of the screen. The "Save" button now has a spinning loading icon.](/assets/images/cloud-index-settings/07-saving.png)
+Once Meilisearch finishes indexing, the primary key will no longer appear in the searchable attributes list:
 
-Once Meilisearch is ready, the button should change to a green color. You will also see a notification pop-up on the right-hand side of your screen.
+![The same "Searchable attributes" list as before. It only contains five searchable attributes after removing the primary key.](/assets/images/cloud-index-settings/07-searchable-attributes-attribute-deleted.png)
 
-![The "Searchable attributes" configuration section showing two buttons at the bottom of the screen. The "Save" button has changed color and now reads "Saved".](/assets/images/cloud-index-settings/08-saved.png)
+If you deleted the wrong primary key, click on "Add attributes" to add it back to the list. You may also click on "Reset to default", which will bring back the searchable list to its original state when you first added your first document to this index:
+
+![The same "Searchable attributes" list as before. Two buttons on its top-right corner are highlighted: "Reset to default" and "Add attributes".](/assets/images/cloud-index-settings/08-searchable-attributes-reset.png)
 
 ## Conclusion
 
 You have used the Meilisearch Cloud interface to check the value of an index setting. This revealed an opportunity to improve your project's performance, so you updated this index setting to make your application better and more responsive.
 
-This tutorial used the searchable attributes setting, but the procedure is the same no matter which index setting you are editing.
+This tutorial used the "Searchable attributes" setting, but the procedure is the same no matter which index setting you are editing.
```

Comment:
@guimachiavelli, what do you think of adding a section that lists all other settings pages somewhere in that page so readers can quickly navigate to understand why they are made for and if they should leverage them?

Reply:
It would be out of place in this page, I think. The goal of this tutorial is primarily to teach people how to navigate the settings UI, with a secondary goal of promoting Cloud usage. Adding descriptions of all existing settings would not help us in either of those goals, I think, and would turn it into more of a reference document.

We can discuss whether the tutorial is actually useful, and whether we need an index settings reference that's not API-centric, but both are outside the scope of this PR, which is mostly about making sure the docs are reflecting the latest cloud updates.